### PR TITLE
MAISTRA-411 renamed product OLM metadata to "servicemesh"

### PR DESCRIPTION
**Pull request by dmitri-d**

### CODE REVIEW

**Review by rcernich — APPROVED** [review]

File: tmp/build/generate-manifests.sh [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
+: ${COMMUNITY:-"false"}
 : ${MAISTRA_VERSION:?"Need to set maistra version, e.g. 1.0.0"}
 
+if [[ ${COMMUNITY} == "true" ]]; then
+  BUILD_TYPE="maistra"
+else
+  BUILD_TYPE="servicemesh"
+fi
+: ${DEPLOYMENT_FILE:="deploy/$BUILD_TYPE-operator.yaml"}
+: ${MANIFESTS_DIR:="manifests-$BUILD_TYPE"}
```
Comment (jwendell) [inline tmp/build/generate-manifests.sh]: For consistency can you use ${BUILD_TYPE} in both lines?

File: tmp/build/generate-manifests.sh [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
-# to generate Maistra OLM metadata: MAISTRA_VERSION=1.0.0 DEPLOYMENT_FILE=deploy/maistra-operator.yaml MANIFESTS_DIR=manifests-maistra tmp/build/generate-manifests.sh
-# to generate ServiceMesh OLM metadata: MAISTRA_VERSION=1.0.0 DEPLOYMENT_FILE=deploy/servicemesh-operator.yaml MANIFESTS_DIR=manifests-servicemesh tmp/build/generate-manifests.sh
+# to generate Maistra OLM metadata: COMMUNITY=true MAISTRA_VERSION=1.0.0 tmp/build/generate-manifests.sh
+# to generate ServiceMesh OLM metadata: MAISTRA_VERSION=1.0.0 tmp/build/generate-manifests.sh
 
 set -e
 
-: ${DEPLOYMENT_FILE:?"Need to set location of the source deployment file, e.g. deploy/maistra-operator.yaml"}
-: ${MANIFESTS_DIR:?"Need to set location of the manifests directory, e.g. manifests/"}
+: ${COMMUNITY:-"false"}
```
Comment (jwendell) [inline tmp/build/generate-manifests.sh]: In docker_build and other scripts the default for this variable is true. Mind to change it for consistency? You would need to change the comments above as well.

File: tmp/build/generate-manifests.sh (hunk 1 above)
Comment (dmitri-d) [reply tmp/build/generate-manifests.sh]: done.

File: tmp/build/generate-manifests.sh (hunk 2 above)
Comment (dmitri-d) [reply tmp/build/generate-manifests.sh]: done.

**Review by jwendell — APPROVED** [review]

**Review by brian-avery — APPROVED** [review]

**Pull request merged**